src/main/java/com/linuxforhealth/connect/builder/BlueButton20RestRouteBuilder.java

```
@@ -48,16 +48,26 @@ public void configure() {
                 .get()
                 .route()
                 .routeId("bluebutton-20-rest-authorize")
-                .process(handleBlueButtonAuth)
-                .toD("${exchangeProperty[location]}");
+                .doTry()
+                    .process(handleBlueButtonAuth)
+                    .toD("${exchangeProperty[location]}")
+                .doCatch(Exception.class)
+                    .setProperty("errorMessage", simple(exceptionMessage().toString()))
+                    .to("direct:error")
+                .end();
```

A bad encryption password causes the blue button routes to fail early, so added try/catch, plus made the LFH message envelop attributes "best effort" as opposed to expecting them to always be set.